src/compare.ts

```diff
@@ -87,20 +87,28 @@ function mutateForBetterStringify(
     newDBTypes.set(type.name, fjss(mutateForBetterStringify(newTypesName, newGroupsName, type)));
   }
 
+  let changed = false;
+
   for (const type of oldDBTypes.keys()) {
     if (!newDBTypes.has(type)) {
       console.log(chalk`{gray [}{red Removed}{gray ]} ${type}`);
+      changed = true;
     }
   }
 
   for (const type of newDBTypes.keys()) {
     if (!oldDBTypes.has(type)) {
       console.log(chalk`{gray [}{green Added}{gray ]} ${type}`);
       console.log(chalk.italic.gray`${newDBTypes.get(type)}`);
+      changed = true;
     } else if (oldDBTypes.get(type) !== newDBTypes.get(type)) {
       console.log(chalk`{gray [}{blue Edited}{gray ]} ${type}`);
       console.log(chalk.gray.italic`Previously:\n${oldDBTypes.get(type)}`);
       console.log(chalk.gray.italic`Now: \n${newDBTypes.get(type)}`);
+      changed = true;
     }
   }
+  if (!changed) {
+    console.log(chalk.green("No changes detected"));
```

Comment:
For consistency with the rest of the chalk calls
```suggestion
    console.log(chalk.green`No changes detected`);
```